### DISCUSSION TITLE
Welcome to daohaus-app Discussions!

### DISCUSSION THREAD

**Discussion by jonathanprozzi** [odyssy-automaton/pokemol-web#1858] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by plor** [comment]
Thanks for setting this up @jonathanprozzi !

**Reply by jonathanprozzi** [reply]
Absolutely! Looking forward to some threads!

**Comment by VaughnWBraswellSr** [comment]
Minion Set-up V3 Dev Suite.  Settings display Minion V0 in the settings.  How can we fix this to actively control our settings and detect crosschain activity? our Dogecoin is not showing up inside the minion.  but it shows on the chronos blockchain it was sent.
